lib/jsdom/living/structured-clone/structuredClone.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+const { structuredDeserializeWithTransfer } = require("./structuredDeserializeWithTransfer");
+const { structuredSerializeWithTransfer } = require("./structuredSerializeWithTransfer");
+
+module.exports = function (globalObject) {
+  return function (value, options = {}) {
+    // Let serialized be ? StructuredSerializeWithTransfer(value, options["transfer"]).
+    const serialized = structuredSerializeWithTransfer(globalObject)(value, options.transfer);
```

Comment:
Factor out `structuredSerializeWithTransfer(globalObject)` into a variable held by the closure, so that it isn't re-created every call.